backend/spherre/app/utils/decorators.py

```
@@ -0,0 +1,13 @@
+from functools import wraps
+
+from flask import jsonify
```

Ruff (0.11.9)

`flask.jsonify` imported but unused

Remove unused import: `flask.jsonify`

(F401)